### PR TITLE
Add Zillow

### PR DESCRIPTION
**Pull request by Rabadash8820**

### CODE REVIEW

File: entries/z/zillow.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Zillow": {
+    "domain": "zillow.com",
+    "tfa": [
+      "sms",
+      "call"
+    ],
+    "documentation": "https://zillow.zendesk.com/hc/en-us/articles/360057432754-2-Step-Verification",
```
Comment (hkamran80) [inline entries/z/zillow.com.json]:
```suggestion
    "documentation": "https://zillow.zendesk.com/hc/articles/360057432754",
```

File: entries/z/zillow.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Zillow": {
+    "domain": "zillow.com",
+    "tfa": [
+      "sms",
+      "call"
```
Comment (hkamran80) [inline entries/z/zillow.com.json:6]: I don't see any mention of calls for 2FA mentioned on the documentation page you linked.

File: img/z/zillow.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg"><path fill="#006AFF" d="M12.006 0 1.086 8.627v3.868c3.386-2.013 11.219-5.13 14.763-6.015.11-.024.16.005.227.078.372.427 1.586 1.899 1.916 2.301a.128.128 0 0 1-.03.195 43.607 43.607 0 0 0-6.67 6.527c-.03.037-.006.043.012.03 2.642-1.134 8.828-2.94 11.622-3.452V8.627zm-.48 11.177c-2.136.708-8.195 3.307-10.452 4.576V24h21.852v-7.936c-2.99.506-11.902 3.16-15.959 5.246a.183.183 0 0 1-.23-.036l-2.044-2.429c-.055-.061-.062-.098.011-.208 1.574-2.3 4.789-5.899 6.833-7.418.042-.03.031-.06-.012-.042Z"/></svg>
```
Comment (hkamran80) [inline img/z/zillow.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21.85 24"><path fill="#006aff" d="M10.93 0 .01 8.63v3.86c3.39-2 11.22-5.12 14.77-6.01.1-.02.16 0 .22.08l1.92 2.3a.13.13 0 0 1-.03.2 43.6 43.6 0 0 0-6.67 6.52c-.03.04 0 .04.01.03a76.73 76.73 0 0 1 11.62-3.45V8.63Zm-.48 11.18c-2.13.7-8.2 3.3-10.45 4.57V24h21.85v-7.94c-2.99.51-11.9 3.16-15.96 5.25a.18.18 0 0 1-.23-.04l-2.04-2.43c-.06-.06-.06-.1.01-.2 1.57-2.3 4.79-5.9 6.83-7.42.05-.03.03-.06 0-.04z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/z/zillow.com.json (hunk 2 above)
Comment (Rabadash8820) [reply entries/z/zillow.com.json:6]: Strange. It is an option in account settings though:

image

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**